CHANGELOG.rst

```diff
@@ -5,6 +5,16 @@ Change Log
    This file loosely adheres to the structure of https://keepachangelog.com/,
    but in reStructuredText instead of Markdown.
 
+2020-08-12
+**********
+
+.. I wish we used a tool to manage changelogs so that I didn't have to choose
+    a date when I made a commit, since I don't know what date the change will
+    be merged...
```

Comment:
And we don't even "release" cookiecutters. I have a canned rant about how we manage versioning in changelogs for repos where we *do* cut releases...